script/hassfest/requirements.py

```diff
@@ -2,11 +2,13 @@
 from __future__ import annotations
 
 from collections import deque
+from functools import cache
```

Comment:
Can you please move the changes to "requirements.py" into a separate PR?
It contains lots of code changes that would be easier to review separately.

Reply:
Split the cache changes to #82124.